database/tbcd/level/level.go

```diff
@@ -209,8 +236,21 @@ func New(ctx context.Context, cfg *Config) (*ldb, error) {
 		}
 	}
 
+	return l, nil
+}
+
+func New(ctx context.Context, cfg *Config) (*ldb, error) {
+	log.Tracef("New")
+	defer log.Tracef("New exit")
+
+	l, err := open(ctx, cfg)
+	if err != nil {
+		return nil, fmt.Errorf("open: %w", err)
+	}
+
 	// Upgrade database
 	for {
+		reopen := false
```

Comment:
nit: can be empty value var
```suggestion
		var reopen bool
```